README.md

```diff
@@ -229,3 +229,9 @@ select * from catalog_iceberg.sales.customers;
 Apache Gravitino is an effort undergoing incubation at The Apache Software Foundation (ASF), sponsored by the Apache Incubator. Incubation is required of all newly accepted projects until a further review indicates that the infrastructure, communications, and decision making process have stabilized in a manner consistent with other successful ASF projects. While incubation status is not necessarily a reflection of the completeness or stability of the code, it does indicate that the project has yet to be fully endorsed by the ASF.
 
 <sub>Apache®, Apache Gravitino&trade;, Apache Hive&trade;, Apache Iceberg&trade;, and Apache Spark&trade; are either registered trademarks or trademarks of the Apache Software Foundation in the United States and/or other countries.</sub>
+
+## Kubernetes
+
+```
+$ helm upgrade --install gravitino-playground ./gravitino-playground --create-namespace --namespace gravitino-playground
```

Comment:
Where is the folder `./gravitino-playground`?  Maybe it comes from `helm package helm-chart` and `tar -xzvf gravitino-playground-0.1.0.tgz` ?

Reply:
fixed, it should be `./helm-chart/`